modules/labs-react/search-form/stories/examples/Basic.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import {MenuItem} from '@workday/canvas-kit-preview-react/menu';
+import {DeprecatedMenuItem} from '@workday/canvas-kit-preview-react/menu';
```

Comment:
I'm assuming we update other components stories that use a soft deprecated component when we do a hard deprecation? I know this is within labs but just curious since this is what is shown in the code examples.

Reply:
Or in this case should we be replacing it with Menu from main now?

Reply:
I guess this should be a part of the hard deprecation ticket for v9.

Reply:
Ok sounds good - we mention you can still use it within the docs so that mean we can also still use it 😜